tests/test_aggregate_metric_for_metrics.py

```diff
@@ -0,0 +1,129 @@
+"""
+Test for functions for aggregating metric for metrics scores
+(src/locomoset/metrics/aggregate_metric_for_metrics.py)
+"""
+
+import numpy as np
+import pytest
+
+from locomoset.metrics.aggregate_metric_for_metrics import aggregate_metric_scores
+
+
+def test_aggregate_metric_for_metrics_perfect_scores_spearmans(
+    dummy_perfect_metric_scores, dummy_validation_scores, test_n_features
+):
+    """Test for aggregation of metric of metrics over datasets or models for perfect
+    metric scores
+    """
+    collected_metric_scores = np.array(
+        [dummy_perfect_metric_scores for _ in range(test_n_features)]
```

Comment:
I'm confused about the use of/need for `test_n_features` in all the tests - can talk it through when we catch up in a bit

Reply:
oh is it taking the role of either number of datasets / number of models? Maybe can rename it somewhere so that's clear.

Reply:
Yeah it's fulfilling that role here, I'll rename it within the tests as a variable for clarity.